scripts/helper-update-version

```diff
@@ -0,0 +1,17 @@
+ #!/bin/bash
+ # Retrieve sources, get sha256sum, parse package version and update these strings in manifest, README and /conf/app.src
+ 
+url=$(curl -s https://api.github.com/repos/duniter/cesium/releases/latest | grep "browser_" | grep "web" | head -1 | cut -d\" -f4)
+wget -nc --quiet $url -P ./tmp
+CHECKSUM=$(sudo sha256sum ./tmp/cesium-*-web.zip | head -c 64)
```

Comment:
Why do you need a `sudo` here?

Reply:
I guess I don't anymore... It's a remaining of earlier tests...

Reply:
Done